sentry_sdk/utils.py

```diff
@@ -34,6 +34,13 @@
 CYCLE_MARKER = object()
 
 
+global_repr_processors = []
+
+
+def add_global_repr_processor(processor):
```

Comment:
I am not sure if this is the right approach, what if someone wants to have a different `repr_processor` for a different client

Reply:
They can dispatch internally based on the currently active client. I don't think attaching this to a client is great since event processors don't work like that either.

We can discuss having them on the scope but I don't imagine that being useful.

Reply:
But that means we would need to have `repr_processors` also on the client, maybe I missed it but I couldn't find this in the pr right now.

Reply:
Why do you think we need this on the client at all? The Python SDK does not have any processors attached to the client.

Reply:
OK, what I mean:

Integrations live on the client, integrations register an `event_processor` (mostly).
The way it works is that `setup_once` registers a `global_event_processor`.

I don't see where the client has any control which `repr_processors` is running?
You are just adding the `_django_queryset_repr` globally which runs for every client.

Reply:
Ah yes sorry, this is missing: I meant to check in `_django_queryset_repr` if the integration is active for the current client.